kotlin/quantize/QuantizerWsmeans.kt

```
@@ -0,0 +1,188 @@
+    if (startingClusters.isNotEmpty()) {
+      clusterCount = min(clusterCount, startingClusters.size)
+    }
+    val clusters = arrayOfNulls<DoubleArray>(clusterCount)
+    var clustersCreated = 0
+    for (i in startingClusters.indices) {
+      clusters[i] = pointProvider.fromInt(startingClusters[i])
+      clustersCreated++
+    }
+    val additionalClustersNeeded = clusterCount - clustersCreated
+    if (additionalClustersNeeded > 0) {
+      for (i in 0 until additionalClustersNeeded) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyForBlock)